eZ/Publish/Core/Persistence/Legacy/Content/Type/MemoryCachingHandler.php

```diff
@@ -416,6 +423,19 @@ public function publish( $contentTypeId )
     }
 
     /**
+     * @see \eZ\Publish\SPI\Persistence\Content\Type\Handler::getFieldMap
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/commit/ea68a7308fbb2d49301821e454850b915d7c7bf0